src/Facade/Services/BomVerificationHistoryFacadeService.cs

```diff
@@ -0,0 +1,84 @@
+﻿namespace Linn.Purchasing.Facade.Services
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq.Expressions;
+
+    using Linn.Common.Domain.Exceptions;
+    using Linn.Common.Facade;
+    using Linn.Common.Persistence;
+    using Linn.Common.Proxy.LinnApps;
+    using Linn.Purchasing.Domain.LinnApps.Boms;
+    using Linn.Purchasing.Resources.Boms;
+
+    public class BomVerificationHistoryFacadeService : FacadeResourceService<BomVerificationHistory, string, BomVerificationHistoryResource, BomVerificationHistoryResource>
+    {
+        private readonly IBuilder<BomVerificationHistory> resourceBuilder;
+
+        private readonly ITransactionManager transactionManager;
+
+        private readonly IDatabaseService databaseService;
+
+        private readonly IBomVerificationHistoryService bomVerificationHistoryService;
+
+        public BomVerificationHistoryFacadeService(
+            IRepository<BomVerificationHistory, string> repository,
+            ITransactionManager transactionManager,
+            IBuilder<BomVerificationHistory> resourceBuilder,
+            IBomVerificationHistoryService bomVerificationHistoryService,
+            IDatabaseService databaseService)
+            : base(repository, transactionManager, resourceBuilder)
+        {
+            this.resourceBuilder = resourceBuilder;
+            this.databaseService = databaseService;
+            this.bomVerificationHistoryService = bomVerificationHistoryService;
+            this.transactionManager = transactionManager;
+        }
+
+        protected override BomVerificationHistory CreateFromResource(BomVerificationHistoryResource resource, IEnumerable<string> privileges = null)
+        {
+            if (resource == null)
```

Comment:
your domain already throws an exception when this thing isn't found, so why repeat that logic here?